lib/npg_tracking/glossary/composition/component/illumina.pm

```diff
@@ -12,6 +12,16 @@ with qw( npg_tracking::glossary::composition::component
 
 our $VERSION = '0';
 
+sub filename {
+  my ($self, $ext) = @_;
+  $ext //= q[];
```

Comment:
Do we need a minumum Perl version of 5.10 (Build.PL or code?) now we're using this operator? Not too fussed....